packages/contracts/src/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "homepage": "https://github.com/aragon/osx#readme",
   "dependencies": {
-    "@aragon/osx-commons-contracts": "1.4.0-alpha.1",
+    "@aragon/osx-commons-contracts": "1.4.0-alpha.3",
```

Comment:
The latest version is `1.4.0-alpha.5`
https://github.com/aragon/osx-commons/releases/tag/1.4.0-alpha.5-contracts

Reply:
This is very strange. We corrected it in https://github.com/aragon/osx/pull/591